packages-next/fields/src/types/image/tests/test-fixtures.ts

```diff
@@ -68,22 +68,20 @@ export const crudTests = (keystoneTestWrapper: any) => {
       keystoneTestWrapper(async ({ context }: { context: any }) => {
         const filenames = ['keystone.jpeg', 'keystone.jpg', 'keystone'];
         for (const filename of filenames) {
-          const data = await createItem({
-            context,
-            listKey: 'Test',
-            item: { avatar: prepareFile(filename) },
-            returnFields: `
-            avatar {
-              id
-              mode
-              filesize
-              width
-              height
-              extension
-              ref
-              src
-            }
-        `,
+          const data = await context.lists.Test.createOne({
```

Comment:
The changes in this PR allow this test to be re-written with the items API, where previously it would fail.